apps/web/Dockerfile

```diff
@@ -22,4 +22,10 @@ COPY pnpm-lock.yaml .
 RUN pnpm install
 RUN pnpm add @babel/core -W
 
-CMD [ "pnpm", "start:docker:web" ]
+COPY .eslintrc.js .
+COPY .prettierrc .
+COPY .prettierignore .
+
+RUN pnpm build
```

Comment:
Can we only `build:web` here? because it's not necessary to build the entire monorepo here.